UI-MVC/Views/Project/Create.cshtml

```diff
@@ -0,0 +1,94 @@
+@using System.Collections
+@using DAL.Data_Transfer_Objects
+@using Domain.Projects
+@using Domain.Users
+@model UIMVC.Models.ProjectViewModel
+@{
+    ViewBag.Title = "Create";
+    Layout = "_Layout";
+
+    IEnumerable<Platform> platforms = (IEnumerable<Platform>) ViewData["platforms"];
+}
+
+<h2>Create Project</h2>
+
+
+<form asp-controller="Project" asp-action="Create" method="post">
+
+    <div class="form-row">
+        <div class="form-group col-md-12">
+            <label asp-for="Project.Title">Naam</label>
+            <input asp-for="Project.Title" class="form-control" value="Test"/>
+        </div>
+        <div class="form-group col-md-6">
+            <label asp-for="Project.StartDate">StartDate</label>
+            <input type="date" asp-for="Project.StartDate" class="form-control" value="1/1/0001 12:00:00 AM"/>
+        </div>
+        <div class="form-group col-md-6">
+            <label asp-for="Project.EndDate">EndDate</label>
+            <input type="date" asp-for="Project.EndDate" class="form-control" value="1/1/0001 12:00:00 AM"/>
+        </div>
+        <div class="form-group col-md-6">
+            <label asp-for="Project.Goal">Hoofdvraag</label>
+            <input asp-for="Project.Goal" class="form-control" value="Test"/>
+        </div>
+        <div class="form-group col-md-6">
+            <label asp-for="Project.Status">Status</label>
```

Comment:
Ofwel status niet editable maken oftewel ervoor zorgen dat deze automatisch in uppercase naar de manager gaat.